install_config/default_imagestreams_templates.adoc

```diff
@@ -0,0 +1,420 @@
+
+[horizontal]
+Languages::
+- link:../using_images/s2i_images/nodejs.html[Node.js]
+- link:../using_images/s2i_images/perl.html[Perl]
+- link:../using_images/s2i_images/php.html[PHP]
+- link:../using_images/s2i_images/python.html[Python]
+- link:../using_images/s2i_images/ruby.html[Ruby]
+Database::
+- link:../using_images/db_images/mongodb.html[MongoDB]
+- link:../using_images/db_images/mysql.html[MySQL]
+- link:../using_images/db_images/postgresql.html[PostgreSQL]
+Other Services::
+- link:../using_images/other_images/jenkins.html[Jenkins]
+endif::[]
+
+ifdef::openshift-origin[]
+Image streams and templates are provided for the following technologies:
+
+[horizontal]
+Languages::
+- link:../../using_images/s2i_images/nodejs.html[Node.js]
+- link:../../using_images/s2i_images/perl.html[Perl]
+- link:../../using_images/s2i_images/php.html[PHP]
+- link:../../using_images/s2i_images/python.html[Python]
+- link:../../using_images/s2i_images/ruby.html[Ruby]
+Database::
+- link:../../using_images/db_images/mongodb.html[MongoDB]
+- link:../../using_images/db_images/mysql.html[MySQL]
+- link:../../using_images/db_images/postgresql.html[PostgreSQL]
+Other Services::
+- link:../../using_images/other_images/jenkins.html[Jenkins]
+endif::[]
+
+ifdef::openshift-enterprise[]
+If you also have the relevant xPaaS Middleware subscription active on your
+account, image streams and templates are also provided and supported by Red Hat
+for each of following middleware services:
+
+[horizontal]
+Middleware Services::
+- link:../using_images/xpaas_images/eap.html[JBoss EAP]
+- link:../using_images/xpaas_images/a_mq.html[JBoss A-MQ]
+- link:../using_images/xpaas_images/jws.html[JBoss Web Server]
+- link:../using_images/xpaas_images/fuse.html[JBoss Fuse Integration Services]
+- link:../using_images/xpaas_images/decision_server.html[Decision Server]
+- link:../using_images/xpaas_images/data_grid.html[JBoss Data Grid]
+endif::[]
+
+You can check which default image streams and templates are currently available
+in your OpenShift environment by logging into the web console and clicking *Add
+to Project*, or getting the lists for the *openshift* project using the CLI:
+
+----
+$ oc get is -n openshift
+$ oc get templates -n openshift
+----
+
+The JSON object definitions used to create these sets are installed on the file
+system of your master during
+ifdef::openshift-enterprise[]
+a link:../install_config/install/quick_install.html[quick] or
+link:../install_config/install/advanced_install.html[advanced installation].
+endif::[]
+ifdef::openshift-origin[]
+an link:../../install_config/install/advanced_install.html[advanced
+installation].
+endif::[]
+If you used another method to install, or if the default image streams and
+templates are ever removed or changed, you can use the following instructions to
+create the default objects yourself. Otherwise, the following instructions are
+not necessary.
```

Comment:
@bparees @adellape 
Agreed this could be clearer. Perhaps we could approach it from another angle. "Run x command to see if you need to do all these things." 
What's the simplest way for an admin to tell if they need to do this or not?